app/controllers/api/console.php

```diff
@@ -111,3 +115,29 @@
 
         $response->chunk('', true);
     });
+
+App::get('v1/console/resources/:resourceId')
+    ->desc('Check resource ID availability')
+    ->groups(['api', 'projects'])
+    ->label('scope', 'projects.read')
```

Comment:
this is not a route that is accessed with an API key right? So there is no scope required

Reply:
I'm not 100% sure about this. Do you think it should be a public endpoint? We can still add abuse checks if it's a public endpoint, right?

Reply:
I checked the code again and this looks correct 👍